.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - '*'
+  push:
+    branches:
+      - master
```

Comment:
shouldn't this be opendistro-0.10?

Reply:
These are trigger. Checkout action will checkout the current PR only